src/main/kotlin/net/irisshaders/lilybot/extensions/util/ThreadInviter.kt

```diff
@@ -43,22 +43,20 @@ class ThreadInviter : Extension() {
 			check { failIf(event.message.author?.id == kord.selfId) }
 
 			action {
-				if (event.guildId == null) return@action
-				val supportTeam = DatabaseHelper.selectInConfig(event.guildId!!, DatabaseManager.Config.supportTeam)
-				val supportChannel = DatabaseHelper.selectInConfig(event.guildId!!, DatabaseManager.Config.supportChannel)
+				val supportTeamId = DatabaseHelper.selectInConfig(event.guildId!!.toString(), "supportTeam")
+				val supportChannelId = DatabaseHelper.selectInConfig(event.guildId!!.toString(), "supportChannel")
 
-				if (supportTeam.equals("NoSuchElemementException") || supportChannel.equals("NoSuchElementException")) {
-					return@action
-				}
-				try {
-					if (event.message.channelId != Snowflake(supportChannel!!)) return@action
-				} catch (e: NumberFormatException) {
-					return@action
-				}
+				// you have to check as a string apparently
+				if (supportChannelId == null || supportTeamId == null) {return@action}
```

Comment:
```suggestion
				if (supportChannelId == null || supportTeamId == null) { return@action }
```
Sorry it irritated me